src/vl-map-layer.js

```diff
@@ -227,12 +255,12 @@ export class VlMapLayer extends VlElement(HTMLElement) {
         });
     }
 
-    __getBoundingbox() {
-        let boundingbox;
+    get __boundingBox() {
+        let boundingBox;
         if (this._source && this._source.getFeatures().length > 0) {
-            boundingbox = this._source.getExtent();
+            boundingBox = this._source.getExtent();
         }
-        return boundingbox;
+        return boundingBox;
```

Comment:
kunt ge niet gewoon if (...) return this._source.getExtent() doen, al dan niet met nen else.
declaratie buiten de if niet nodig lijkt mij.

Reply:
Heb ik niet geschreven, dus ken de reden niet. Ik vermoed dat getExtent() niet altijd proper werkt zonder features in de source?

Reply:
Ik zal het versimpelen naar 

```
if (this._source && this._source.getFeatures().length > 0) {
  return this._source.getExtent();
}
```

Aan de if-test kom ik for now liever niet aangezien ik niet weet wat daar de reden voor was.